GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -26,6 +27,8 @@ class AccountDetailsViewController: UIViewController {
            let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String {
             versionAndBuildLabel.text = "Version: \(versionNumber), Build: \(buildNumber)"
         }
+        
+        setupCrashlyticsHiddenButton()
```

Comment:
why do you need hidden button for Crashlytics?

Reply:
In order to test that it is working, it will be removed after we see it is working